runners/flink/src/main/java/org/apache/beam/runners/flink/LyftFlinkStreamingPortableTranslations.java

```diff
@@ -76,7 +78,7 @@
 
 public class LyftFlinkStreamingPortableTranslations {
 
-  private static final Logger logger =
+  private static final Logger LOG =
```

Comment:
Had to rename as was recommended by Checkstyle. 
`./gradlew :runners:flink:1.10:spotlessJavaCheck`  and `./gradlew :runners:flink:1.10:spotlessJavaApply`

Reply:
Yes, all `static final` fields have to be capitalized.